docs/reference/8-admin-app/getting-started/installation/older-versions-of-admin-app/admin-app-for-suite-3-v201.md

```diff
@@ -0,0 +1,199 @@
+
+1. Configure `$dbConnectionInfo`. These values are used to construct the connection strings.
+    1. `Server`. The name of the database server. For a local server, we can use "(local)" for SQL and "localhost" for PostgreSQL.
+
+    2. `Engine.` Admin App supports SQL and PostgreSQL database engines. So setting up the `Engine` will decide which database engine to be used. Valid values are "SQLServer" and "PostgreSQL".
+    3. `UseIntegratedSecurity.` Will either be "$true" or "$false".
+        1. If you plan to use Windows authentication, this value will be "$true"
+        2. If you plan to use SQL Server/ PostgreSQL server authentication, this value will be "$false" and the Username and `Password` must be provided.
+    4. `Username`. Optional. The username to connect to the database. If `UseIntegratedSecurity` is set to $true, this entry is not needed
+    5. `Password`. Optional. The password to connect to the database. If `UseIntegratedSecurity` is set to $true, this entry is not needed
+    6. `Port.` Optional. Used to specify the database server port, presuming the server is configured to use the specific port.
+2. Configure `$adminAppFeatures`. These values are used to set Optional overrides for features and settings in the web.config.
+    1. `ApiMode.` Possible values: `sharedinstance`, `districtspecific` and `yearspecific`. Defaults to `sharedinstance`
+    2. `SecurityMetadataCacheTimeoutMinutes`. Optional. Defaults to 10 minute security metadata cache timeout.
+3. Configure `$p`. This is the variable used to send all the information to the installation process.
+    1. `ToolsPath`. Path for storing installation tools, e.g., nuget.exe. Defaults to "C:/temp/tools"
+    2. `OdsApiUrl`. Base URL for the ODS / API. Mandatory.
+    3. `AdminDatabaseName`. , `OdsDatabaseName`, `SecurityDatabaseName`. Optional. Specify _only_ if ODS / API was set with a custom database name.
+        1. For example, when configuring the `OdsDatabaseName`, the value here will be the name of the ODS database, whereas the `AdminDatabaseName` and `SecurityDatabaseName` will be the name of the Admin and Security databases, respectively.
+    4. `WebApplicationName`, `WebApplicationPath`. Optional. Defaults to "AdminApp" and "C:\\inetpub\\Ed-Fi\\AdminApp" respectively.
+    5. `WebSitePort`. Optional. Defaults to 443.
+    6. `WebsiteName`. Optional. Defaults to "Ed-Fi".
+    7. `PackageVersion`. Optional. If not set, will retrieve the latest full release package.
+
+Below is an example of the configuration of the "install.ps1" file for SQL Server with `sharedinstance` mode:
+
+**install.ps1(SQLServer)** Expand source
+
+```
+$dbConnectionInfo = @{
+ Server = "(local)"
+ Engine = "SqlServer"
+ UseIntegratedSecurity = $true
+}
+
+$p = @{
+ DbConnectionInfo = $dbConnectionInfo
+ OdsApiUrl = "https://localhost:54746"
+}
+```
```

Comment:
Need to remove "Expand Source"

Here's a case of a weird Confluence export problem. This code is short enough that I wouldn't bother necessarily with using `<details>...</details>` to create an expand region.

This is a really good opportunity for using tabs for SQL Server / PostgreSQL content. You would need to rename the page with `.mdx` as the extension, and then search the repo for "<TabItem>" to see an example.

Not _necessary_ though, and could be done later.  One more tip: remove `**install.ps1(SQLServer)**` and add a title to the code block

~~~
```powershell title="install.ps1 for SQL Server"
~~~

Reply:
"** Expand source" does not appear in a search and resolved.  Thanks!